test/test_instrument/NDB_BVL_Instrument_testtest.class.inc

```diff
@@ -42,6 +42,9 @@ class NDB_BVL_Instrument_testtest extends NDB_BVL_Instrument
 
         // setup the form
         $this->_setupForm();
+
+	// set data entry type
+	$this->DataEntryType = 'DirectEntry';
```

Comment:
something is wrong with the indentation here. could you fix it ? @jesscall

Reply:
@ridz1208 done

Reply:
These still show up as bad indent and I dont think they should be there at all

Reply:
not sure why it's not working, hold on